### ISSUE TITLE
`sprockets-htmlimports` URL broken?

### ISSUE THREAD

**Issue by liubin** [nevir/polymer-rails#2]
Hi

In the README.md, I can't find the sprockets-htmlimports (https://github.com/nevir/sprockets-htmlimports), nor the 

```
gem 'sprockets-htmlimports', git: 'https://github.com/nevir/sprockets-htmlimports'
```

Is it changed to https://github.com/nevir/hackish-sprockets-htmlimports?

Thanks.

**Comment by nevir** [comment]
Yeah, sorry - the hackish-htmlimports is what it used to be. I'm currently
working on rebuilding it to make use of Sprockets' upcoming support for
content type negotiation, but it's a ways out.

Unfortunately that means I'm ignoring polymer-rails until then :( Stay
tuned!
